### PR TITLE
Arch Migrator

### PR DESCRIPTION
**Pull request by chrisburr**
See #9

### CODE REVIEW

**Comment by sodre** [comment]
@chrisburr, is popt arch migrator ready to merge?

**Comment by chrisburr** [comment]
@sodre yes it is.

#9 can then be closed.

File: recipe/build.sh [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 set -e -o pipefail
 
+curl -L 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD' > config.guess
+curl -L 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD' > config.sub
```
Comment (sodre) [inline recipe/build.sh]: Can we convert these two curls into entries in the source section with sha256sums?

Also, shouldn't we be pulling the config{.guess,sub} of the same version of popt? In your URL it looks like we are downloading from the master branch.

**Review by sodre — CHANGES_REQUESTED** [review]
Please move them

File: recipe/build.sh (hunk 1 above)
Comment (chrisburr) [reply recipe/build.sh]: Yes, sorry I meant to do that and forgot.

The error message I got without them suggested to use these URLS and the master branch.

Comment (sodre) [reply recipe/build.sh]: Ahh, I see. The files exist already. Shouldn't we do a patch instead in that case?

Comment (chrisburr) [reply recipe/build.sh]: I've committed a suggested approach which I think is cleaner as it's easier to update if required. If you'd prefer a patch let me know and I'll do it that way.

Comment (sodre) [reply recipe/build.sh]: I like your solution better 👍

**Comment by chrisburr** [comment]
Thanks for the prompt review/merge! 🍰

**Pull request merged**